app/store/models/LendableObject.d.ts

```diff
@@ -0,0 +1,30 @@
+import type { RoleType } from 'app/utils/constants';
+import type { EntityId } from '@reduxjs/toolkit';
+import type { Duration } from 'moment-timezone';
+
+interface LendableObject {
+  id: EntityId;
```

Comment:
```suggestion
  id: ID;
```
`import type { ID } from 'app/store/models'`

Reply:
Outdated